hyperspy/_components/eels_cl_edge.py

```diff
@@ -93,6 +93,9 @@ def __init__(self, element_subshell, GOS=None):
                             'fine_structure_coeff',
                             'effective_angle',
                             'onset_energy'])
+
+        self._whitelist['element_subshell'] = ('init', element_subshell)
+        self._whitelist['GOS'] = ('init', GOS)
```

Comment:
It looks like it should. `fine_structure_width` as well, don't know why I did not add them. Will add to https://github.com/hyperspy/hyperspy/pull/167 , where it belongs. By the way, should I PR that again (to get rid of the old discussions), or is it fine like that?